base/checks.c

```diff
@@ -764,6 +767,16 @@ int handle_async_service_check_result(service *temp_service, check_result *queue
 		temp_service->acknowledgement_type = ACKNOWLEDGEMENT_NONE;
 		temp_service->notified_on = 0;
 
+		/* we log any change from SOFT to HARD type state happening during the rest */
```

Comment:
[...] to HARD type state -> [...] to HARD state type

Reply:
Done